OMS/Enum/FileStatus.cs

```diff
@@ -4,12 +4,13 @@
 using System.Text;
 using System.Threading.Tasks;
 
-namespace OMS
+namespace Enum
```

Comment:
change name space to OMS.Models.Enums
and move all enum files into OMS.Models project